src/views/Restoration/RestorationReviewConfirm.vue

```diff
@@ -27,16 +33,18 @@
       <header>
         <h2>Document Delivery</h2>
         <p class="mt-4">
-          Copies of the incorporation documents will be sent to the email addresses listed below.
+          Copies of the restoration documents will be sent to the email addresses listed below.
         </p>
       </header>
 
       <v-card flat class="mt-6">
         <DocumentDelivery
           class="py-8 px-6"
-          :contactValue="getBusinessContact.email"
-          :completingPartyEmail="getUserEmail"
-          contactLabel="Registered Office"
+          contactLabel="Business Office"
+          :contactValue="businessOfficeEmail"
+          additionalLabel="Applicant"
+          :additionalValue="applicantEmail"
+          :showCompletingParty="false"
```

Comment:
Note that "false" needs to be a bound prop. This doesn't work with `showCompletingParty="false"`.